pkg/kubelet/util/manager/cache_based_manager_test.go

```diff
@@ -385,7 +385,7 @@ func podWithSecretsAndUID(ns, podName, podUID string, toAttach secretsToAttach)
 func TestCacheInvalidation(t *testing.T) {
 	fakeClient := &fake.Clientset{}
 	fakeClock := testingclock.NewFakeClock(time.Now())
-	store := newSecretStore(fakeClient, fakeClock, noObjectTTL, time.Minute)
+	store := NewObjectStore(getSecret(fakeClient), fakeClock, noObjectTTL, time.Minute)
```

Comment:
@TommyStarK Can you explain this change? There are multiple calls of `newSecretStore` in this file. Changing only one of them looks confusing to me.

Reply:
IIRC I wanted to cover the getSecret function, but I am not sure. I can revert that part if you want.

Reply:
you can introduce new test to cover it instead. That would be clear and explicit.

Reply:
I've reverted that change, I think I made a mistake

Reply:
/lgtm